adobecom/milo
# Helix 5 support for milo blocks and features

Re-raise of https://github.com/adobecom/milo/pull/3122 from adobecom rather than my fork

### Description
- There are a few special cases, such as loc, where I'm unsure about implications of moving certain links to aem.page, hence those simply got a "TODO" for now.
- There are other special cases, such as unit tests, where currently porting to `aem.page` does not make any sense, as our default is `hlx.page` unless detected otherwise. Some other unit tests *needed* to be adapted.
- This should *not* change any production behavior and development behavior apart of potentially pulling content from `.aem.` (federal/milo) rather than `.hlx.`

### Testing instructions
both `https://main--milo--adobecom.aem.page/docs/library/kitchen-sink/` and `https://main--milo--adobecom.hlx.page/docs/library/kitchen-sink/` should work and have no differences.

Consuming pages should also not have any difference in experience: https://main--cc--adobecom.hlx.live/de/products/photoshop?milolibs=hlx5-upgrade--milo--adobecom

Lastly, all tools should also be checked (caas, preflight, loc) to ensure we didn't accidentally introduce any errors.

Resolves: [MWPW-157482](https://jira.corp.adobe.com/browse/MWPW-157482)

**Test URLs:**
- Before: https://main--milo--adobecom.aem.page/?martech=off
- After: https://hlx5-upgrade--milo--adobecom.aem.page/?martech=off
